gufe/storage/resultstore.py

```diff
@@ -0,0 +1,63 @@
+import warnings
+
+from typing import ClassVar
+
+from gufe.storage.errors import (
+    MissingExternalResourceError, ChangedExternalResourceError
+)
+
+
+class ResultStore:
```

Comment:
This class name is very similar to ResultsClient and might be better renamed to something else

Reply:
Changed this to `ResultServer`